apps/condo/domains/organization/utils/helpers.ts

```diff
@@ -27,11 +27,20 @@ export const roleToQuery = (rolesIds: Array<string>): OrganizationEmployeeRoleWh
     }
 }
 
-export const roleToSearchQuery = (roleName: string): OrganizationEmployeeRoleWhereInput => {
+export const roleToSearchQuery = (search: string, translates) => {
```

Comment:
Why do you use the verb `translates` for an array of translation strings O_o?
```suggestion
export const roleToSearchQuery = (search: string, translations) => {
```